pkg/platform/yaml/testdb/db.go

```diff
@@ -134,6 +134,9 @@ func (ts *TestYaml) upsert(ctx context.Context, testSetID string, tc *models.Tes
 	if err != nil {
 		return tcsInfo{name: tcsName, path: tcsPath}, err
 	}
+
+	data = append([]byte(utils.GetVersionAsComment()), data...)
```

Comment:
You should also check whether the test file exists because this function is also used when the test case is updated.